### ISSUE TITLE
Warning: redefining `object_id' may cause serious problems

### ISSUE THREAD

**Issue by zoras** [twalpole/apparition#8]
```[bash]
❯ bin/rspec spec
Running via Spring preloader in process 71021
/ruby/2.6.2/lib/ruby/gems/2.6.0/bundler/gems/apparition-dcd9d74b34cf/lib/capybara/apparition/dev_tools_protocol/remote_object.rb:52: warning: redefining `object_id' may cause serious problems
```

**Comment by twalpole** [comment]
Thanks for catching this -- fix will be pushed shortly

**Comment by twalpole** [comment]
Fixed via 62d94b03fd7d5a340d65e244f46eb51e0c8f9d37
